android-lib/build.gradle

```diff
@@ -54,19 +54,27 @@ android {
     abortOnError false
   }
   packagingOptions {
+    exclude 'META-INF/NOTICE'
+    exclude 'META-INF/LICENSE'
     exclude 'META-INF/maven/com.google.guava/guava/pom.properties'
     exclude 'META-INF/maven/com.google.guava/guava/pom.xml'
   }
 
+  dokka {
+    outputFormat = 'javadoc'
+    jdkVersion = 8
+    outputDirectory = file("../../../devcenter/content/reference/android-v3.0/api/android-lib")
+    sourceDirs = files('src/main')
+  }
 }
 
 configurations {
-  jaxDoclet
-  classpaths
   compile.exclude module: 'guava-jdk5'
 }
 
 buildscript {
+  ext.kotlin_version = '1.3.40'
+  ext.dokkaVersion = '0.9.17'
```

Comment:
if the dokka also applies for the java-core project better to move the the `parent` gradle file

Reply:
Updated.